Core/Graphics/Include/Babylon/Android/GraphicsPlatform.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#include <android/native_window.h>
+
+using WindowType = ANativeWindow*;
```

Comment:
If we do it this way, I think there are a couple implications:
1. There can only be one window type per platform that must be known at compile time. E.g. the same compilation can't use both `HWND` and `SwapChainPanel*`. I'm not sure if this is a problem or not.
2. I don't think there is any reason to even have Graphics be templated if we do it this way.

If we think that each compilation might need to support multiple window types, then I would think we would want to keep the template, and provide one or more template specializations per platform (for each window type supported by that platform). Thoughts @bghgary?

Reply:
Regarding 1) i was planning on handing cases where we may want multiple window types via defining our generic type with a variant, and using reflection to change behavior based off the current type at runtime, which may or may not be optimal.

I'm also a bit wary of whether or not we need our templated Graphics class as well, I'm interested in @syntheticmagus's opinion regarding what we're hoping to accomplish using it.